orchestrators/Orchestrator.py

```diff
@@ -134,18 +136,24 @@ def generateFilterString(self, userToken):
     def format_as_ndjson(self, obj: dict) -> str:
         return json.dumps(obj, ensure_ascii=False) + "\n"
 
+    def parse_multi_columns(columns: str) -> list:
+        if "|" in columns:
+            return columns.split("|")
+        else:
+            return columns.split(",")
+
     # Format request body and headers with relevant info based on search type
     def prepare_body_headers_with_data(self, request):
         request_messages = request.json["messages"]
 
         body = {
-            "messages": request_messages,
-            "temperature": float(self.AZURE_OPENAI_TEMPERATURE),
-            "max_tokens": int(self.AZURE_OPENAI_MAX_TOKENS),
-            "top_p": float(self.AZURE_OPENAI_TOP_P),
-            "stop": self.AZURE_OPENAI_STOP_SEQUENCE.split("|") if self.AZURE_OPENAI_STOP_SEQUENCE else None,
-            "stream": self.SHOULD_STREAM,
-            "dataSources": []
+        "messages": request_messages,
```

Comment:
The previous indent level was correct on these props.

Reply:
Fixed starting on new line 150.